IFC4x3/Properties/m/MechanicalBendingStrength_20vAwvkL95e8aB2n$YHbS$/Documentation.md

```diff
@@ -0,0 +1 @@
+The bending force that the indulator could withstand.
```

Comment:
???

Reply:
Need @fhulin opinion

Reply:
Like we already discussed, Needed for mechanical purpose!

Reply:
BEndingStrength ok! 94e95c3b013d4bc02e88d41ccea2f2c71b7a79b9